src/Compute/Compute/ChangeLog.md

```diff
@@ -23,7 +23,8 @@
 * Fixed typo 'DeyAll' to 'DenyAll' in NetworkAccessPolicy description. 
 * Preannounced breaking changes. Please refer to https://go.microsoft.com/fwlink/?linkid=2333229
 * Updated New-AzDiskConfig to include `SecurityMetadataUri` and `SecurityDataUri` parameters.
-* Updated Grant-AzDisk to return `SecurityMetadataAccessSAS` property.
+* Updated Grant-AzDiskAccess to return `SecurityMetadataAccessSAS` property.
+* Updated Grant-AzSnapshotAcess to return `SecurityMetadataAccessSAS` property.
```

Comment:
please move this line right under `upcoming release`, it helps better resolve conflict